packages/rpc-cli/README.md

```diff
@@ -0,0 +1,72 @@
+# Platformatic RPC CLI
+
+Fastify CLI command to generate a server RPC api for a Fastify application.
+
+> [!WARNING]  
+> Platformatic RPC API is in the experimental stage. The feature is not subject to semantic versioning rules.
+> Non-backward compatible changes or removal may occur in any future release.
+> Use of the feature is not recommended in production environments.
+
+## Installation
+
+```bash
+npm install @platformatic/rpc-cli
+npm install --save-dev @platformatic/rpc-cli
+```
+
+## Usage
+
+1. Register the plugin in your Fastify typescript application. Pass the OpenAPI schema as an option.
+OpenAPI schema will be automatically generated by the CLI command.
+
+```ts
+import { join } from 'node:path'
+import { readFile } from 'node:fs/promises'
+import fastify from 'fastify'
+import fastifyRpc from '@platformatic/rpc'
+
+const fastify = fastify()
+
+const openapiSchemaPath = join(__dirname, 'openapi.json')
+const openapiSchemaFile = await readFile(openapiSchemaPath, 'utf8')
+const openapiSchema = JSON.parse(openapiSchemaFile)
+
+fastify.register(fastifyRpc, {
+  openapi: openapiSchema,
+  prefix: '/rpc',
+})
+```
+
+2. Define RPC handlers in your Fastify application. An RPC handler is a function that takes an optional options object.
+All parameters will be passed as properties of the options object.
+
+```ts
+type User = {
+  name: string
+  age: number
+}
+
+const users = [
+  { name: 'Alice', age: 30 },
+  { name: 'Bob', age: 25 },
+  { name: 'Charlie', age: 35 },
+]
+
+fastify.rpc('getUsers', async (options: { maxAge: number }): Promise<User[]> => {
+  return users.filter(user => user.age <= options.maxAge)
+})
```

Comment:
Can you please make this example complete? I'd expect this would be inside a Fastify plugin with a few imports. This example needs fixing on the other README as well.

Maybe just reference the `rpc` docs from the `rpc-cli` one?